var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -50,79 +51,105 @@ class NeurodamusModel(Package):
     _hoc_srcs = ('common/hoc', 'hoc')
     _mod_srcs = ('common/mod', 'mod')
 
-    @staticmethod
-    def copy_all(src, dst, copyfunc=shutil.copy):
-        isdir = os.path.isdir
-        for name in os.listdir(src):
-            pth = join_path(src, name)
-            isdir(pth) or copyfunc(pth, dst)
+    # The name of the mechanism, which cen be overriden
+    mech_name = ""
 
     def merge_hoc_mod(self, spec, prefix):
+        core_prefix = spec['neurodamus-core'].prefix
         # First Initialize with core hoc / mods
-        copy_tree(spec['neurodamus-core'].prefix.hoc, '_merged_hoc')
-        copy_tree(spec['neurodamus-core'].prefix.mod, '_merged_mod')
+        copy_tree(core_prefix.hoc, '_merged_hoc')
+        copy_tree(core_prefix.mod, '_merged_mod')
+
+        # If we shall build mods for coreneuron, only bring from core those specified
+        if spec.satisfies("+coreneuron"):
+            mkdirp('core_mechs')
+            with open(core_prefix.mod.join("coreneuron_modlist.txt")) as core_mods:
+                for aux_mod in core_mods:
+                    shutil.copy(core_prefix.mod.join(aux_mod.strip()), 'core_mechs')
 
```

Comment:
May be we can look today at these mod files and avoid this step by making mod files coreneuron compatible?

Reply:
Is it worth to change? I think in the medium term coreneuron should be able simply to skip those files which are not important

Reply:
ok..I dont want to delay the merge...go ahead with current setup.